lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -402,6 +413,9 @@ def load_device_info(device_info)
           var_value.merge! value
         when Array
           var_value << value
+        when Integer, String
+          @logger.info("Replacing #{var} value #{var_value} -> #{value}")
```

Comment:
Is this log really needed? We don't emit logs when overwriting values for `Hash`.

Reply:
removed